code/index.html

```diff
@@ -4,18 +4,34 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>Hamsters are the best</title>
```

Comment:
😆

Reply:
Woops, was suppose to change that, just a placeholder text 😆  But it's true, they are the best!